wakuv2/waku.go

```diff
@@ -478,13 +480,13 @@ func (w *Waku) identifyAndConnect(ctx context.Context, isLightClient bool, peerI
 	case <-w.ctx.Done():
 		return
 	case <-w.identifyService.IdentifyWait(conns[0]):
-		if isLightClient {
-			err = w.node.Host().Network().ClosePeer(peerInfo.ID)
-			if err != nil {
-				w.logger.Error("could not close connections to peer", zap.Stringer("peer", peerInfo.ID), zap.Error(err))
-			}
-			return
-		}
+		// if isLightClient {
+		// 	err = w.node.Host().Network().ClosePeer(peerInfo.ID)
+		// 	if err != nil {
+		// 		w.logger.Error("could not close connections to peer", zap.Stringer("peer", peerInfo.ID), zap.Error(err))
+		// 	}
+		// 	return
+		// }
```

Comment:
Why is this commented?

Reply:
When debugging the tests previously, this logic makes the connected peer always 0.
But now it actually not the case. I will bring it back.
I'm wondering, even the peer is close here, a light client can still connect the closed peers later which is stored in peer store manager, right?